package.json

```diff
@@ -95,7 +95,7 @@
     "codelyzer": "^6.0.1",
     "commitizen": "^4.2.3",
     "cz-conventional-changelog": "^3.3.0",
-    "husky": "^5.1.2",
+    "husky": "^4.3.8",
```

Comment:
is this downgrade intentional?

Reply:
Yeah. v5 is not free and requires some additional setup to make it work. So we reverted this change.